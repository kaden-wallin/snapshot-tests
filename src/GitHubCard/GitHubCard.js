import React from "react"
import Card from "react-bootstrap/Card"
import Avatar from "./avatar-icon.png"

function GitHubCard() {
    return (
        <div className="card">
          <Card style={{ width: "18rem"}}>
          <Card.Img variant="top" src={Avatar} />
          <Card.Body>
              <Card.Title>kaden-wallin</Card.Title>
              <Card.Text>
                
JS-Stock-Charts Public

JS-Encapsulation-Practice Public

JS-Building-Targeted-Ads Public

5.7.6 activity starter

JS-Building-an-Interactive-Map Public

JS-Jacks-Paint Public

JS-Alliance-Brand Public

Activity for 5.6.2

JS-Web-Game-Part-7 Public

JS-Asynchrony-With-Callback-Functions Public

5.5.5-activity-array-automotive Public

JS-Web-Game-Part-6 Public

JS-Callback-Practice Public

JS-Web-Game-Part-2 Public

JS-Web-Game-Part-1 Public

CSS-Final-Project Public

RR-Mocking-HTTP-Requests

              </Card.Text>
          </Card.Body>
          </Card>
        </div>
    );
  }

  export default GitHubCard