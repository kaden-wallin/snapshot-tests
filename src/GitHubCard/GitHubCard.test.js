import GitHubCard from './GitHubCard'
import renderer from 'react-test-renderer'

test('renders a snapshot', () => {
  const tree = renderer.create(<GitHubCard />).toJSON()
  expect(tree).toMatchSnapshot()
})