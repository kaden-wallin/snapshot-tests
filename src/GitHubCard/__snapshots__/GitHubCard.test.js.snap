// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div
  className="card"
>
  <div
    className="card"
    style={
      Object {
        "width": "18rem",
      }
    }
  >
    <img
      className="card-img-top"
      src="avatar-icon.png"
    />
    <div
      className="card-body"
    >
      <div
        className="card-title h5"
      >
        kaden-wallin
      </div>
      <p
        className="card-text"
      >
        JS-Stock-Charts Public JS-Encapsulation-Practice Public JS-Building-Targeted-Ads Public 5.7.6 activity starter JS-Building-an-Interactive-Map Public JS-Jacks-Paint Public JS-Alliance-Brand Public Activity for 5.6.2 JS-Web-Game-Part-7 Public JS-Asynchrony-With-Callback-Functions Public 5.5.5-activity-array-automotive Public JS-Web-Game-Part-6 Public JS-Callback-Practice Public JS-Web-Game-Part-2 Public JS-Web-Game-Part-1 Public CSS-Final-Project Public RR-Mocking-HTTP-Requests
      </p>
    </div>
  </div>
</div>
`;
